src/main.cpp

```diff
@@ -16,6 +16,11 @@
 #include <Arduino.h>
 #include <Hoymiles.h>
 #include <LittleFS.h>
+#include "OledDisplay.h"
+

```

Comment:
All diese Variablen können direkt in der OledDisplay.h bzw. cpp definiert werden. Es sollte nur einen include der header file geben und anschließend einen .init aufruf.